### ISSUE TITLE
Basic Video editing 

### ISSUE THREAD

**Issue by lianergoist** [FossifyOrg/Gallery#152]
### Checklist

- [X] I made sure that there are **no existing issues** - [open](https://github.com/FossifyOrg/Gallery/issues) or [closed](https://github.com/FossifyOrg/Gallery/issues?q=is%3Aissue+is%3Aclosed) - to which I could contribute my information.
- [X] I made sure that there are **no existing discussions** - [open](https://github.com/FossifyOrg/Gallery/discussions) or [closed](https://github.com/FossifyOrg/Gallery/discussions?discussions_q=is%3Aclosed) - to which I could contribute my information.
- [X] I have read the FAQs inside the app (Menu -> About -> FAQs) and my problem isn't listed.
- [X] **I have taken the time to fill in all the required details. I understand that the bug report will be dismissed otherwise.**
- [X] This issue contains only one feature request.
- [X] I have read and understood the [contribution guidelines](https://github.com/FossifyOrg/Gallery/blob/master/CONTRIBUTING.md).
- [X] I optionally [donated](https://fossify.org/donate) to support the Fossify mission.

### Feature description

I think it would be nice if we could do some basic video editing, like trimming start and end (maybe even without re-encoding), rescale to reduce size, maybe add watermark or text, etc. 

### Why do you want this feature?

When I open a video in Gallery, I am given the option to edit it. But when I try to do so, I am told there is no valid app to do that. 

### Additional information

I think it should be possible to find a free video editing library that would make it relatively simple to add this editing feature. ffmpeg comes to mind, but there might be other and maybe better alternatives.

**Comment by Aga-C** [comment]
https://github.com/FossifyOrg/Gallery/discussions/90

**Comment by Aga-C** [comment]
@lianergoist Why have you closed that feature request? It's not a duplicate of what I've linked above, it was just a discussion about reverting video and image editors.

**Comment by lianergoist** [comment]
Okay, I'll reopen it.

**Comment by bjlockie** [comment]
I get a (Fossify) Gallery Basic Editor in v1.1.3

**Comment by Philippe-Choquette** [comment]
Trim and crop and the only things I would need, as it can be useful with screen recordings. Having this capability built-in would be better than having to find a third-party app.

**Comment by bjlockie** [comment]
Same, all I use is crop to be able to share small parts.

**Comment by OnyxJet** [comment]
Yes, we need a video editor! Nothing too complicated. Just trimming, cropping, maybe even text and filters. That sort of thing.

**Comment by Cwpute** [comment]
In the meantime, you can use Open Source Video Editor for simple edits on Android here : https://f-droid.org/packages/io.github.devhyper.openvideoeditor/

**Comment by colvdv** [comment]
Back when I used Simple Mobile Tools Gallery I was able to crop videos easily. I switched to Fossify since Simple Mobile Tools doesn't work on my new phone. Is it possible for someone to take the code from Simple Mobile Tools Gallery and add it to Fossify Gallery with ease? The apps are basically identical in look and feel, after all.

**Comment by bjlockie** [comment]
I think fossify gallery crops but doesn't do annotating...
I think the simplemobile code for that does that is closed source.

**Comment by Mr-Bajs** [comment]
Some basic video editing functionality is most wanted.

**Comment by iam-amazing** [comment]
Video trimming is needed.
